region/maridia/inner-yellow.json

```diff
@@ -879,7 +879,8 @@
                 {
                   "name": "Base",
                   "notable": false,
-                  "requires": []
+                  "requires": [ "canConsecutiveWalljump" ],
+                  "devNote": "This should require something more to kill the Choot."
```

Comment:
I would say `canPreciseWallJump` or `Screw Attack` or `Power Bomb` or `enemy hit`
There is a fair sized safe spot on the ledge, and you can continue walljumping when it floats towards that side.
Wave is too gimicky with the camera manip you have to do.
Firing a strong weapon just before landing would be a precise jump and some kind of quickFire tech